### PR TITLE
updated deployment script and reviews.php

### PR DESCRIPTION
**Pull request by lkthang**
Reviews.php uses sql queries and sends an update when customer wants to submit a review.
Deployment script adds reviews to be out of 5 rather than 10

### CODE REVIEW

File: Deployment_Script.sql [hunk 1]
```diff
@@ -180,8 +180,7 @@ CREATE TABLE `Picture` (
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8;
 
 
-
-INSERT INTO `Picture` (`Pid`, `URL`, `Dish`) VALUES
+NSERT INTO `Picture` (`Pid`, `URL`, `Dish`) VALUES
```
Comment (chasestubb) [inline Deployment_Script.sql]: bro. 😆 

you deleted the "I"

File: Deployment_Script.sql [hunk 2]
```diff
@@ -343,3 +342,14 @@ BEGIN
 	END IF;
 END;
 $$
+
+drop trigger if exists CreateNewReview
```
Comment (chasestubb) [inline Deployment_Script.sql:346]: nice

File: Reviews.php [hunk 3]
```diff
@@ -47,158 +79,60 @@
...
-                ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat
-                nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
-            </div>
-          </card>
-        </div>
-      </div>
+
+		<?php
+			//Populate page with reviews
+  		   $query = "SELECT Review.*, Customer.Name FROM Review, Customer WHERE Review.Cid = Customer.Cid ORDER BY ReviewRating DESC";
```
Comment (chasestubb) [inline Reviews.php:96]: nicely done.

**Review by chasestubb — COMMENTED** [review]
line 183 and you're good

File: Deployment_Script.sql (hunk 1 above)
Comment (lkthang) [reply Deployment_Script.sql]: lol. ill fix dat

**Pull request merged**